### PR TITLE
Update free-courses-en.md

### PR DESCRIPTION
**Pull request by webcleric**
Just thought that this freeCodeCamp.org could fit in the list :)

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

Adding resources

### Why is this valuable (or not)?

It's comprehensive and free

### How do we know it's really free?

Uploaded to Youtube

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [ ] Search for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction)

## Followup

- Check the output of Travis-CI for linter errors!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -393,6 +393,7 @@
 * [learn:query](https://learnquery.infinum.co)
 * [Namaste 🙏 JavaScript: An In-Depth JavaScript Fundamentals Course](https://www.youtube.com/playlist?list=PLlasXeu85E9cQ32gLCvAvr9vNaUccPVNP) - Akshay Saini (YouTube)
 * [Programming Foundations with Javascript, HTML and CSS](https://www.coursera.org/learn/duke-programming-web) - Owen Astrachan, Robert Duvall, Andrew D. Hilton, Susan H. Rodger (Coursera)
+* [Learn JavaScript - Full Course for Beginners](https://www.youtube.com/watch?v=PkZNo7MFNFg) - freeCodeCamp.org
```
Comment (davorpa) [inline courses/free-courses-en.md:396]: Completes main creator.

```suggestion
* [Learn JavaScript - Full Course for Beginners](https://www.youtube.com/watch?v=PkZNo7MFNFg) -  Beau Carnes, freeCodeCamp.org
```

**Review by davorpa — CHANGES_REQUESTED** [review]
From video description...

> This is a complete 134-part JavaScript tutorial for beginners will teach you everything you need to know to get started with the JavaScript programming language.
> 
> ⭐️Curriculum⭐️
> This is a stand-alone video but it follows the JavaScript curriculum at freecodecamp.org. Access the curriculum here:
> 🔗 Basic JavaScript: https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/
> 🔗 ES6 JavaScript: https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/es6/
> 
> ⭐️Code⭐️
> This course was created using scrimba.com. Access the course there along with the code:
> 🔗 Basic JavaScript: https://scrimba.com/playlist/pny4ghw
> 🔗 ES6 JavaScript: https://scrimba.com/playlist/p7v3gCd

We prefer playlist Youtube videos, although this seems be a single video with timeline chapters covering freecodecamp's JS curriculum path

Also remember [check linter errors](https://github.com/EbookFoundation/free-programming-books/pull/5539/checks?check_run_id=3605241630#step:7:1).
```log
Run fpb-lint ./courses/
/home/runner/work/free-programming-books/free-programming-books/courses/free-courses-en.md
  382:1-396:113  warning  Alphabetical ordering: swap l.396 and l.395  alphabetize-lists  remark-lint
```
So reorder items to complain list alphabetization

**Comment by eshellman** [comment]
Be sure to check the output of Travis-CI for linter errors. Alphabetical order and spacing are the most common problems.

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -393,6 +393,7 @@
 * [learn:query](https://learnquery.infinum.co)
```
Comment (LuigiImVector) [inline courses/free-courses-en.md:393]: a

**Comment by davorpa** [comment]
@eshellman remember close/reject this. it's fixed by #5566

**Pull request closed without merging**